### ISSUE TITLE
Visually selecting strings with `lighter` variant is next to impossible

### ISSUE THREAD

**Issue by cideM** [marko-cerovac/material.nvim#37]
Visual selection and string highlighting don't mix well

![Screenshot from 2021-06-03 09-39-27](https://user-images.githubusercontent.com/4246921/120606583-a125c680-c44f-11eb-9b91-bf322c9521d3.png)

**Comment by marko-cerovac** [comment]
This is a problem with the color palette itself. I made this theme following the official [material theme palette](https://www.material-theme.com/docs/reference/color-palette/) and the lighter variant is just poorly written in terms of visibility.
If you look at the original material theme for VSCode, it also has the same issue with strings.

![vscode](https://user-images.githubusercontent.com/76592799/120647239-ab10ef00-c47a-11eb-8183-9fc962bbaa47.png)

That's why a higher contrast version of the light theme was added. If you put ```vim.g.material_lighter_contrast = true``` in your vimrc, the lighter theme will be much more visible. 

![2021-06-03-14:56:47-screenshot](https://user-images.githubusercontent.com/76592799/120647982-76e9fe00-c47b-11eb-885a-cdc490628068.png)


This was done like this because I want to keep the theme as true to the original as possible, but I also want it to be as visible as possible, so I added these two variations of the lighter theme so that users can choose between the original and the more visible one.
